src/AnimalList/AnimalList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Animal from '..//Animal/Animal';
+
+export default function AnimalList({ animals }) {
+  return (
+    <main>
+      {animals.map((animal) => 
+        (<Animal
+          key={animal}
+          animal={animal} />
```

Comment:
here you could use a spread operator! 
so `<Animal
          key={animal}
          {...animal}/>